templates/includes/webp-img.jade

```diff
@@ -0,0 +1,35 @@
+mixin webp-img(src, title, alt, picture_style, img_style)
+  - var path_nosuffix = src.replace(/[.][a-z]+/,"");
+  - var absolute_path = (path.resolve("./contents"+src.match(/\/media.+/, "")));
+  - var absolute_nosuffix = absolute_path.replace(/[.][a-z]+/,"");
+
+
```

Comment:
I would suggest adding some code comments to document the mixin so that you (and others) know what is going on here in the future.